### ISSUE TITLE
Can't install ICE currently, CU 01 error.

### ISSUE THREAD

**Issue by Malorn-Deslor** [AsteriskAmpersand/Ice-Stable#19]
So, I can't even install ICE now, claims it can't find CU 01 repeatedly.

`2024-10-22 19:08:45,172: Installing Stable
2024-10-22 19:08:51,548: Generating Symlink [cG0]
2024-10-22 19:08:51,549: Generating Symlink [cG1]
2024-10-22 19:08:51,549: Generating Symlink [cG10]
2024-10-22 19:08:51,549: Generating Symlink [cG11]
2024-10-22 19:08:51,550: Generating Symlink [cG2]
2024-10-22 19:08:51,550: Generating Symlink [cG3]
2024-10-22 19:08:51,551: Generating Symlink [cG4]
2024-10-22 19:08:51,551: Generating Symlink [cG5]
2024-10-22 19:08:51,551: Generating Symlink [cG6]
2024-10-22 19:08:51,552: Generating Symlink [cG60]
2024-10-22 19:08:51,552: Generating Symlink [cG7]
2024-10-22 19:08:51,552: Generating Symlink [cG8]
2024-10-22 19:08:51,553: Generating Symlink [cG9]
2024-10-22 19:08:51,557: Downloading Missing CU 01 [1eRllU2Ba8Rhj1QX-Uv6MEy5XhqjAkhJK]
2024-10-22 19:08:51,557: Downloading CU cG20.bin [1eRllU2Ba8Rhj1QX-Uv6MEy5XhqjAkhJK]
2024-10-22 19:08:51,947: Failed to Download Missing CU 01 [1eRllU2Ba8Rhj1QX-Uv6MEy5XhqjAkhJK]`

I made sure that Stracker's was up to date, just to be sure. Checked for leftover files as instructed. I had ICE installed and completely functional last time I played a few months back.

Anything obvious I'm missing here?

**Comment by Malorn-Deslor** [comment]
Yes, I am missing the fact that all of ICE is not compatible with the current version of monster hunter. Which might be good to mention somewhere rather than having to find out by going through the discord carefully.

Maybe put this information something so that downpatching is explained as part of the process.

**Comment by Vongola505** [comment]
did you find a solution or do we need to wait for it to fixed?

**Comment by Malorn-Deslor** [comment]
I had to downpatch, instructions on are the discord.
